backend/Node/routes/route-helpers/spotify-helpers/spotify-helpers.ts

```diff
@@ -45,9 +45,9 @@ const MLServerAPIs = {
 	recommendPlaylist: `${apiPrefixML}recommend_playlist`	
 }
 export const SpotifyAPIs = {
-	tracks: `${spotifyAPIPrefix}tracks?ids=%trackIds%`,
```

Comment:
is the `spotifyAPIPrefix` not right?
have you tried adding `NODE_ENV = 'localhost'` in the .env file?

Reply:
spotifyAPIPrefix is correct, but it got duplicated in the URL for the top tracks API call — I've just reverted the changes to SpotifyAPIs but removed the duplication in the other spot, see new commit and comment below